lib/templates.py

```diff
@@ -33,11 +34,13 @@ def __str__(self):
             desc += "\n" + self.reference_url()
         return desc
 
-    def download(self, engine=None, debug=False):
+    def download(self, engine=None, debug=False, populate=False):
+        """when initializing the selected engine create_db=false will not overwrite the existing db"""
```

Comment:
Looks like the name of the argument and that described in the docstring don't match.